website/docs/reference/snapshot-configs.md

```diff
@@ -150,11 +150,27 @@ Configurations can be applied to snapshots using the [YAML syntax](/docs/build/s
 
 </Tabs>
 
+### Snapshot configuration migration
+
+The latest snapshot YAML configurations introduced in dbt v1.9 (such as [`snapshot_meta_column_names`](/reference/resource-configs/snapshot_meta_column_names) and [`dbt_valid_to_current`](/reference/resource-configs/dbt_valid_to_current)) are best suited for new snapshots. For existing snapshots, we recommend the following to avoid any inconsistencies in your snapshots:
```

Comment:
I think it's a short enough list we can just name them all - `snapshot_meta_column_names`,  `dbt_valid_to_current`, `hard_deletes`

Reply:
Also, you say "For existing snapshots, we recommend the following" but then go into a list for new snapshots?